OntologyMapper/src/OntologyMapper/OntologyMappingManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="OntologyMappingManager.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.SmartFacilities.OntologyMapper
+{
+    using Microsoft.Azure.DigitalTwins.Parser;
+    using Microsoft.Azure.DigitalTwins.Parser.Models;
+
+    public class OntologyMappingManager : IOntologyMappingManager
+    {
+        private OntologyMapping ontologyMapping;
+
+        public OntologyMappingManager(IOntologyMappingLoader mappingLoader)
+        {
+            ontologyMapping = mappingLoader.LoadOntologyMapping();
+        }
+
+        public bool TryGetInterfaceRemapDtmi(Dtmi inputDtmi, out DtmiRemap? dtmiRemap)
+        {
+            dtmiRemap = ontologyMapping.InterfaceRemaps.FirstOrDefault(r => r.InputDtmi == inputDtmi.ToString() && !r.IsIgnored);
+
+            if (dtmiRemap != null)
+            {
+                return true;
+            }
+
+            return false;
```

Comment:
This seems backwards? Shouldn't successfully retrieving it return true?

Reply:
Looks ok to me... tests pass

Reply:
Whoops, my bad. I was reading `dtmiRemap == null`